src/pages/access-recovery-confirm/lib/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './map-errors';
```

Comment:
Please, do not create anything except model inside page. Page should contain only `index.tsx` and `model.ts`, nothing else.

You can place error mappings inside `index.tsx` because it affects view-layer.